### ISSUE TITLE
Defining CoC for Stories

### ISSUE THREAD

**Issue by skfantoni** [alan-turing-institute/AutisticaCitizenScience#475]
## Summary

Working out what is appropriate to post in a story.

## Date Due by

TBC

## Details

* Agree on what counts as ableist or offensive re autism
* Decide what counts as inappropriate re race, genre, sexuality etc.
* Agree on how much of the story should be specifically about sensory issues
* Decide how specific people can be in re to places and people

## Suggestions for tasks that can fix this issue

* [ ] Discuss this issue with group at meet-up
* [ ] Create a red-list of words/terms re autism
* [ ] Create a red-list of words/terms re other discrimination 
* [ ] Write up first draft CoC for Stories
* [ ] Review decisions with a larger group
* [ ] Finalise CoC for Stories

## Who can help

Anyone interested in the moderation process

**Comment by GeorgiaHCA** [comment]
YAY!!! Amazing work on your first issue @skfantoni this is great 🎉

**Comment by GeorgiaHCA** [comment]
There is already a lot of excellent discussion relating to this topic. Much of this is in the latest summaries doc. which I'm currently working on - but it's also in previous focus group sessions

**Comment by GeorgiaHCA** [comment]
https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20190918_summary-18-September-2019.md#4-moderation

**Comment by GeorgiaHCA** [comment]
https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20190924_summary-24-September-2019.md#4-moderation

**Comment by GeorgiaHCA** [comment]
https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20191004_summary-4-October-2019.md#4-moderation

**Comment by GeorgiaHCA** [comment]
The above are comments by a diverse group of autistic people and their parents and carers which should be used to co-create the code of conduct

**Comment by skfantoni** [comment]
Here is the earliest of early drafts of the red amber list. After a discussion with James last night I've included two levels of amber: Amber and Yellow. Both are based on context but one is suitable only to over 15s. 

[https://docs.google.com/document/d/1slUYIdOEzr25LXev5U82COSnYjNl3BxefR-dev5c7t8/edit?usp=sharing](url)

**Comment by skfantoni** [comment]
Suggested tasks for Thursday's Meet-Up

* [ ] Discuss age split with the group
* [ ] Review what is already on the list
* [ ] Add the group's suggestions to the list

**Comment by JamesSCTJ** [comment]
Hey @skfantoni. Thanks fordoing this,  I can't seem to open the google doc?

**Comment by lottycoupat** [comment]
@JamesSCTJ @skfantoni When I clicked directly on the link it opened as an issue in github, but when I copied and pasted the link into my browser it worked 😄

**Comment by JamesSCTJ** [comment]
Hi Susanna. Thanks for your work,  I don't have access to track changes in the document, but two key thoughts from me:

-Can we just say the N word rather than removing a character? I think it's such a sensitive word and I mindful that we don't want to upset anyone. I leave it to your judgement on the Y word as I you;'ll be more aware of the context than me. I think we should also be clear before people join the group that there will be sensitive words so people can make that choice.

-Could you also make it clear in the issues box that we're primarily thinking about people's experiences of these topics, not actually content!

Thanks!
James

-

**Comment by skfantoni** [comment]
@JamesSCTJ Yeah I wasn't comfortable typing that to be honest. Not sure what you mean in your second point. I've changed the settings so anyone with the link can now edit the Google Doc. 

[https://docs.google.com/document/d/1slUYIdOEzr25LXev5U82COSnYjNl3BxefR-dev5c7t8/edit?usp=sharing](url)

**Comment by skfantoni** [comment]
One big question is if we are only allowing users above 18, does that mean we are only allowing stories about over 18s? Obviously, this can't be agreed upon until we decide what the situation with second-hand reports are, however, I'll add it into the decisions to make issue too.

**Comment by skfantoni** [comment]
- [ ] Once the initial CoC has been written, seeing how neurotypical and atypical people interpret it is important. If there are discrepancies we need to know what they are in order to improve.

**Comment by GeorgiaHCA** [comment]
Heya @skfantoni ! Would you say this is complete now that we have a CoC written?
